Module '"fs"' has no exported member 'Dirent'.

node_modules/merge-graphql-schemas/typings/index.d.ts:2:10 - error TS2305: Module '"fs"' has no exported member 'Dirent'.
 import { Dirent } from 'fs';

what happened with this guys?

--- a/typings/index.d.ts
+++ b/typings/index.d.ts
@@ -1,5 +1,4 @@
 import { DocumentNode, Source } from 'graphql';
-import { Dirent } from 'fs';
 
 export function mergeTypes(
   types: Array<string | Source | DocumentNode>,
@@ -15,4 +14,4 @@ export function fileLoader(
     extensions?: string[];
     globOptions?: object;
   }
-): Array<string | Buffer | Dirent>;
+): Array<string | Buffer>;

